OPHD/States/MapViewState.cpp

```diff
@@ -458,11 +458,11 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 			break;
 
 		case NAS2D::EventHandler::KeyCode::KEY_PAGEUP:
-			mMapView->moveView(MapOffsetUp);
+			changeViewDepth(0, MapOffsetUp);
 			break;
 
 		case NAS2D::EventHandler::KeyCode::KEY_PAGEDOWN:
-			mMapView->moveView(MapOffsetDown);
+			changeViewDepth(0, MapOffsetDown);
```

Comment:
Rather than changing the call here, I think maybe we need to add an extra call to the proposed `onChangeDepth` event method.